libp2p/src/builder/phase.rs

```diff
@@ -16,17 +16,21 @@ mod websocket;
 
 use bandwidth_logging::*;
 use bandwidth_metrics::*;
+pub use behaviour::BehaviourError;
 use behaviour::*;
 use build::*;
 use dns::*;
 use libp2p_core::{muxing::StreamMuxerBox, Transport};
 use libp2p_identity::Keypair;
+pub use other_transport::TransportError;
 use other_transport::*;
 use provider::*;
 use quic::*;
 use relay::*;
 use swarm::*;
 use tcp::*;
+#[cfg(all(not(target_arch = "wasm32"), feature = "websocket"))]
+pub use websocket::WebsocketError;
```

Comment:
Nit: can we order public items below the private ones?

Reply:
I would love to but it seems that `cargo fmt` won't let me. It always reorder them. And I believe the formatting test won't pass.

Reply:
Even when leaving one empty line between the two blocks?

Reply:
I've tried, the answer is no for some reason.